### ISSUE TITLE
Convert app management add/edit user modal to a page

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-core#8189]
**What feature do you want to improve?**
If you go to App Management > Users > Add (or click on a row to Edit) a modal dialog is shown. Modals aren't good for forms because it restricts usability, eg: you can't hit back, you can't refresh, you can't open in a new tab, screen real estate is restricted unnecessarily, scrolling doesn't behave quite right, etc.

**Describe the improvement you'd like**
Convert the modal into a page with a unique URL.

**Describe alternatives you've considered**
- Fake it by making the modal bigger and looking like a page. This doesn't solve all the accessibility issues above.
- Leave it as is.

**Additional context**

![Screenshot from 2023-04-19 13-52-00](https://user-images.githubusercontent.com/2064938/232946073-c204d836-3efc-4119-b2a3-c05e18ace2ac.png)

**Comment by isis-meng** [comment]
Hi, I attempted to push some changes to a new branch but I don't appear to have access. Is there something I should do to correct my setup? Thanks!

![image](https://user-images.githubusercontent.com/59855227/233191725-1518d8c1-19c7-4132-ab73-eeb072628519.png)

**Comment by garethbowen** [comment]
@isis-meng I suggest you make a git fork of the cht-core repo into your own user, then you'll be able to create your own branches. Then when it's ready you can PR from your fork to the `medic` fork.

**Comment by isis-meng** [comment]
Thanks! I've opened a PR but it's still a work-in-progress -- I've left a comment with some questions I have.

**Comment by AbeebProPlus** [comment]
Hi. I would love resolve this issue if it has not been resolved

**Comment by garethbowen** [comment]
Hi @AbeebProPlus ! Thanks for reaching out. @isis-meng has already started working in the above PR - how are you getting on with this issue?

**Comment by AbeebProPlus** [comment]
It's just a dent on the UI. I hope it's fixed soon enough.

**Comment by garethbowen** [comment]
No response on the PR review so removing from the milestone.

@AbeebProPlus If you felt like picking up where @isis-meng left off that'd be great!

**Comment by Horlawale-ui** [comment]
Hello @garethbowen This PR seems to have been stale
If the PR still needs resolving, can I pick it up??

**Comment by mrjones-plip** [comment]
Thanks for your interest in this ticket @Horlawale-ui !  I've just assigned you [another ticket](https://github.com/medic/cht-core/issues/10240).  Let's get that one well under way before starting a second ticket.  Once that is ready to merge, we'll be sure to send this one your way as well.

**Comment by VirajBhagat** [comment]
Hi, just wanted to check if anyone is currently working on this issue, or should I pick it up and resolve it?
